stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/options/message/internal/MessageOptionsView.kt

```diff
@@ -59,7 +63,11 @@ internal class MessageOptionsView : FrameLayout {
     private fun configureMineMessage(configuration: Configuration, syncStatus: SyncStatus) {
         val iconsTint = configuration.iconsTint
 
-        binding.replyTV.configureListItem(configuration.replyIcon, iconsTint)
+        if (configuration.replyEnabled) {
```

Comment:
Would be nice to extract this into a method and reuse inside `configureTheirsMessage`